tool/lib/commands/release_helper.dart

```diff
@@ -108,19 +108,25 @@ class ReleaseHelperCommand extends Command {
 
       print("Setting the release version.");
       await processManager.runProcess(
-        CliCommand.from('devtools_tool', [
+        CliCommand.from('dart', [
+          'run',
+          'devtools_tool.dart',
```

Comment:
don't we expect `devtools_tool` to be activated in other scripts?

Reply:
for some reason it wasn't working for me and was preventing me from running the helper.
Even though the alias was set it was saying something about a file not existing.
I thought it was weird too since the command was being run with the alias, then inside it was having a hard time finding it

Reply:
does it work if you add 
`workingDirectory: pathFromRepoRoot("tool"),` to both of those commands?

Reply:
this is what I have my alias set to
![Screenshot 2023-09-27 at 11 33 29 AM](https://github.com/flutter/devtools/assets/1386322/dd5e96e8-d9c6-498d-9fe7-f0e697ff456c)

Reply:
To fix my issue I created a script called `devtools_tool` and put it in my ~/bin directory, which I have on my $PATH.
This helps us avoid the issue where aliases don't seem to play well with the dart command execution stuff
```
#!/bin/bash
dart run /Users/danchevalier/development/devtools/tool/bin/devtools_tool.dart "$@"

```